t/plugin/limit-count4.t

```diff
@@ -171,3 +171,57 @@ passed
     }
 --- response_body
 ["1","0","0"]
+
+
+
+=== TEST 5: modified limit-count.incoming, cost == 2
+--- config
+    location = /t {
+        content_by_lua_block {
+            local limit_count_local = require "apisix.plugins.limit-count.limit-count-local"
+            local lim = limit_count_local.new("plugin-limit-count", 10, 60)
+            local uri = ngx.var.uri
+            for i = 1, 7 do
+                local delay, err = lim.limit_count:handle_incoming(uri, 2, true)
```

Comment:
Please add test cases to test the incoming function separately, limit_count.handle_incoming is not a module function of apisix.

Reply:
In fact, the `handle_incoming` is a part of APISIX. I extended the existing function from lua-resty-limit-traffic.

https://github.com/apache/apisix/blob/f294304b61f104db6ef3cff9cc26e9a36d125824/apisix/plugins/limit-count/limit-count-local.lua#L19-L43

Reply:
I know, we hook it in a very hacky way. But essentially it doesn't belong to apisix, if you only test it, some logic in the incoming function will be missed

Reply:
Addressed the issue, please see if the tests are up to the mark.